src/test/scala/workflow/WorkflowUtilsSuite.scala

```diff
@@ -57,7 +57,7 @@ class WorkflowUtilsSuite extends FunSuite with LocalSparkContext with Logging {
       (plusDataPlusLabelEstimator, trainData, labelData)
 
     val initialPipeline = new Optimizer {
```

Comment:
Gotta say - the ability to dynamically construct different optimizers with `::` is awesome.